sdk/tests/deqp/functional/gles3/es3fShaderPrecisionTests.js

```diff
@@ -337,6 +336,9 @@ goog.scope(function() {
 		for (var testNdx = 0; testNdx < this.m_numTestsPerIter; testNdx++) {
 			/** @type {number} */ var in0 = this.m_rnd.getFloat(this.m_rangeA[0], this.m_rangeA[1]);
 			/** @type {number} */ var in1 = this.m_rnd.getFloat(this.m_rangeB[0], this.m_rangeB[1]);
```

Comment:
These random float64s are used in the reference computation. But `highp` is only 32 bits. So this causes `highp_mul` to fail on my workstation.

Reply:
Please add this to the comment below explaining why the random float64s have to be truncated to float32s.